qa/zcash/create_benchmark_archive.py

```diff
@@ -226,8 +226,17 @@ def create_benchmark_archive(blk_hash):
 
     # Make reproducible archive
     os.remove('%s/LOG' % db_path)
+    find = subprocess.Popen(['find', 'benchmark'], stdout=subprocess.PIPE)
+    sort = subprocess.Popen(['sort'], stdin=find.stdout, stdout=subprocess.PIPE)
+    tar = subprocess.Popen([
+        'tar', '--no-recursion',
+        '--owner=0', '--group=0',
+        '--mtime=2017-05-17T00:00:00Z',
+        '-c', '-T', '-'], stdin=sort.stdout, stdout=subprocess.PIPE)
+    archive = subprocess.check_output(['gzip', '-9n'], stdin=tar.stdout)
```

Comment:
Note that if we change this, we need to update CI again, because it expects `.gz` when sending to the worker.